projects/storefrontapp-e2e-cypress/cypress/helpers/checkout-forms.ts

```diff
@@ -1,3 +1,10 @@
+//TODO: find better place for this type
+export type DeepPartial<T> = T extends object
```

Comment:
let's move it to separate file under helpers dir

Reply:
Type has been moved to `form.ts` file under helpers dir